Squashed upstream commits b7154bb a8e7904

Otherwise, .otf fonts are mistaken for OpenDocument Formula files.

diff --git a/freedesktop.org.xml.in b/freedesktop.org.xml.in
index c3f9963..486280d 100644
--- a/freedesktop.org.xml.in
+++ b/freedesktop.org.xml.in
@@ -1885,6 +1885,7 @@ command to generate the output files.
   </mime-type>
   <mime-type type="application/x-font-otf">
     <_comment>OpenType font</_comment>
+    <sub-class-of type="application/x-font-ttf"/>
     <generic-icon name="font-x-generic"/>
     <magic priority="50">
       <match type="string" value="OTTO" offset="0"/>
-- 
cgit v0.10.2
